srcpkgs/rdma-core/template

```diff
@@ -24,21 +26,18 @@ conf_files="
  /etc/rdma/modules/roce.conf
  /etc/rdma/modules/srp_daemon.conf"
 
-post_install() {
-	# Fix path of udev rule
-	mv ${DESTDIR}/usr/etc/udev/rules.d/70-persistent-ipoib.rules \
-	   ${DESTDIR}/usr/lib/udev/rules.d
-
-	# The cmake is bad
-	mv ${DESTDIR}/usr/etc/* ${DESTDIR}/etc/
+post_patch() {
+	# install manpages not depending on pandoc and rst2man
+	sed -i buildlib/rdma_man.cmake \
+		-e 's|"\${I}" MATCHES "\\\\\.md\$"|& AND PANDOC_FOUND|' \
+		-e 's|"\${I}" MATCHES "\\\\\.in\\\\\.rst\$"|& AND RST2MAN_FOUND|'
+}
```

Comment:
Don't abuse `post_patch` for dynamic patching that doesn't need to be dynamic, especially not with `sed` which will happily and silently do nothing when your patterns cease to apply. Make a proper patch.